projects/distributed-tracing/src/shared/dashboard/widgets/waterfall/waterfall/waterfall-chart.module.ts

```diff
@@ -12,8 +12,9 @@ import { WaterfallChartComponent } from './waterfall-chart.component';
     CommonModule,
     TableModule.withCellParsers([SpanNameTableCellParser]),
     TableModule.withCellRenderers([SpanNameTableCellRendererComponent]),
-    TooltipModule
+    TooltipModule,
+    IconModule
   ],
-  exports: [WaterfallChartComponent]
+  exports: [WaterfallChartComponent, SpanNameTableCellRendererComponent]
```

Comment:
This shouldn't need to be exported since we don't directly reference renderers - did you run into an issue?

Reply:
I guess, It was a mistake.
Fixed